runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/harness/StreamingWorkerHarness.java

```diff
@@ -25,4 +25,6 @@ public interface StreamingWorkerHarness {
   void start();
 
   void shutdown();
+
+  long getAndResetThrottleTime();
```

Comment:
ditto could make StreamingWorkerHarness implement the ThrottlingTimerInterface (or whatever you name it).

Reply:
done